.changelog/4114.yml

```diff
@@ -0,0 +1,6 @@
+changes:
+- description: A new version of validate is out, for more information please refer to the [docs](docs-site TBA)
+  type: feature
+- description: The new validate flow is activated automatically while the old one is disabled, for more information on how to switch them back, please refer to the [docs](docs-site TBA)
```

Comment:
lets include info how to switch back in the RN. Also we should add a link to the notes @dorschw is adding (wherever that ends up)

Reply:
Will do.